_posts/2023-03-03-distopia.md

```diff
@@ -0,0 +1,358 @@
+---
+layout: post
+title: Introducing Distopia
+---
+
+Calculating Euclidian distances under periodic boundary conditions is a core component in both running and analysing molecular simulations. For example, the calculation of radial distribution functions, hydrogen bonds, structure factors, and many other common analyses require calculation of one or more pairwise Euclidian distances.
+
+The Euclidian distance formula under periodic boundary conditions can be expressed as:
+
+$$
+R = \sqrt{A}
+$$
+
+Many molecular mechanics engines have highly optimised code for calculating distances that often make use of specialised code paths and/or accelerators to achieve maximal performance. However this functionality is often heavily "baked-in" and is not able to be used as a standalone component.
+
+We at MDAnalysis wanted to bring some of this blazing speed to our distance calculations without sacrificing the modularity of MDAnalysis by introducing heavyweight dependencies on a simulation engine.
```

Comment:
Say early on how to use it in MDA and from which release onwards it is available. 

The detail below is really informative and it's great to have technical blog posts. I would just make sure that our general users get the message that (1) we have fast distance calculations and (2) you can use them right away.